docs/Guide/application-commands/application-command-registry/registering-chat-input-commands.mdx

```diff
@@ -3,56 +3,174 @@ sidebar_position: 3
+external source that using a difference programming language from JavaScript/TypeScript (i.e. an external API or
+database).
 
-This is an optional that defaults to `true`. If set to `false`, the command will not be registered if it is not already
-registered i.e. if not already present in the `Discord API`.
+:::
 
-### behaviorWhenNotIdentical
+### `behaviorWhenNotIdentical`
 
 This options tells the registry that how to behave when the command is not identical to the one which is already
-registered on `Discord API`. By default it's set to [`LOG_TO_CONSOLE`][behavior-log] which means that it will warn you
-about the data differences found as compared to the data received from the `Discord API`.
+registered on Discord API. By default it's set to [`LOG_TO_CONSOLE`][behavior-log] which means that it will only warn
+you about the data differences found as compared to the data received from the Discord API.
 
-If you want the registry to `Overwrite` the command data, you can set this option to [`OVERWRITE`][behavior-overwrite].
+If you want the registry to be overwritten with your own local data, you can set this option to
+[`OVERWRITE`][behavior-overwrite]. Note that configuring [`OVERWRITE`][behavior-overwrite] will also still log the
+change to the console. That is to say, [`OVERWRITE`][behavior-overwrite] includes [`LOG_TO_CONSOLE`][behavior-log]
+rather than both being separate code branches.
 
-### idHints
+### `registerCommandIfMissing`
 
-These are the unique command ids from the `Discord API` that the registry uses to identify the command in case of a name
-change which prevents the recreation of commands accidentally. This is also helpful in delivering the command
-interaction to the correct command class.
+This is an option that defaults to `true`. If set to `false`, the command will not be registered if it is not already
+registered i.e. if not already present in the Discord API.
 
-When `Sapphire` registers a application command it will log the command id for the command in console with the help of
-default logger provided by `Sapphire`. The logging also depends above the `LogLevel` configuration and the
-`behaviorWhenNotIdentical` value which is explained above.
+:::info
 
-To setup logging please checkout [`Configuring Log Level Guide`][configure-logging].
+You will likely never have to touch this option, but it can be useful if you are not using Sapphire to register your
+commands and instead you're using something external to do so.
+
+:::
 
 ## Chat Input Command Registry Methods
 
 :::warning
 
-To register a chat input command, you need to acquire the `ApplicationCommandRegistry`. Please see the
-[`ApplicationCommandRegistry`](./acquiring-application-command-registry) section for more details.
+To register a chat input command, you need to acquire the [`ApplicationCommandRegistry`][acr]. Please see the
+[`ApplicationCommandRegistry`][aaacr] section for more details.
```

Comment:
Two sections?

Reply:
Yes, the top one is the docs and the latter is the guide.